api/models/fixtures/multi_balancer.json

```diff
@@ -91,10 +91,28 @@
         "Ref": "WebPort80Balancer"
       }
     },
+    "WebPort80BalancerName": {
+      "Value": {
+        "Ref": "AWS::StackName"
```

Comment:
If DNSName to ELB name is not straight foward, then your strategy of copying the LoadBalancerName generator for the output is good. Thanks for looking into it.